.devcontainer/devcontainer.json

```diff
@@ -9,8 +9,12 @@
         "vscode": {
             "extensions": [
                 "ms-dotnettools.csdevkit",
-                "ms-vscode.vscode-typescript-next"
+                "ms-vscode.vscode-typescript-next",
+                "ms-azuretools.vscode-bicep"
             ]
         }
+    },
+    "features": {
+        "ghcr.io/azure/azure-dev/azd:latest": {}
```

Comment:
This doesn't work. `azd` is installed, but when I try to run there is an error about docker not being installed 🙃